lib/src/spec/common.dart

```diff
@@ -1,4 +1,4 @@
-import 'package:ably_flutter_plugin/src/spec/spec.dart';
+import 'package:ably_flutter/src/spec/spec.dart';
```

Comment:
Not sure how I missed this earlier, but linting should have picked up [prefer_relative_imports](https://dart-lang.github.io/linter/lints/prefer_relative_imports.html)

Reply:
This can be ignored in this PR and I can take it up later. I'll be checking regularly for any linting pointers for every commit.

Reply:
Also, in upcoming PR's we are capturing these lint pointers in workflows: https://github.com/ably/ably-flutter/blob/b917e0e86fdd5d575b3ab2b57d0646580d00992f/.github/workflows/flutter_integration.yaml

Reply:
I just checked, it was fixed in another branch later: https://github.com/ably/ably-flutter/blob/feature/realtime-presence-get-history/lib/src/spec/common.dart